_install_guide/dns_and_ssl.md

```diff
@@ -42,19 +42,20 @@ In order to enable SSL we'll need to provide Gate (and thus Java) with certifica
 
 For the following example we're going to create self-signed certificates although for production use you will want to have an official certificate authority (CA) sign your certificate.
 
-You can either create a self signed cert, or import your own cert.
+You can **either** import your own cert **or** create a self signed cert:
 
-- Creating a **self-signed CA** with java keystore
+ - Importing **your own Certificate**.
 ```
-apt-get install openjdk-7-jre-headless -y;
-keytool -genkey -keyalg RSA -alias server -keystore keystore.jks -storepass ${YOUR_KEYSTORE_PASSWORD} -validity 360 -keysize 2048
+keytool -importkeystore -srckeystore server.p12 -srcstoretype pkcs12 -srcalias spinnaker -srcstorepass ${YOUR_KEY_PASSWORD} -destkeystore keystore.jks -deststoretype jks -destalias server -deststorepass ${YOUR_KEY_PASSWORD} -destkeypass ${YOUR_KEYSTORE_PASSWORD}
 ```
 
- - Import **your own Certificate**
+- Creating a **self-signed CA (EXPIRES IN 360 DAYS)** with java keystore.
```

Comment:
can you change "expires in 360 days" to a "note" with ">" right below the header?

Reply:
hm... looks a bit ugly, are you sure?
![](https://cl.ly/0P0y0z283O2H/Image%202017-12-26%20at%2010.56.03%20AM.png)

Reply:
agreed, leave it the way you had it.